plugins/modules/mysql_user.py

```diff
@@ -366,6 +373,7 @@ def main():
     ssl_ca = module.params["ca_cert"]
     check_hostname = module.params["check_hostname"]
     db = ''
+    db_context = module.params["db_context"]
```

Comment:
```suggestion
    if module.params["db_context"]:
        db = 'mysql'
```
to keep it closer, would make things more readable